go/client/fuse_status_osx.go

```diff
@@ -0,0 +1,96 @@
+// +build darwin
+
+package client
+
+import (
+	"fmt"
+	"os"
+
+	keybase1 "github.com/keybase/client/go/protocol"
+	"github.com/keybase/go-kext"
+)
+
+func KeybaseFuseStatus(bundleVersion string) keybase1.FuseStatus {
+	status := keybase1.FuseStatus{
+		BundleVersion: bundleVersion,
+		InstallStatus: keybase1.InstallStatus_UNKNOWN,
+		InstallAction: keybase1.InstallAction_UNKNOWN,
+	}
+
+	var kextInfo *kext.Info
+
+	// Check osxfuse 3.x
+	path3 := "/Library/Filesystems/osxfuse.fs"
```

Comment:
My comment was blown away, but did you see it earlier, that this block and the subsequent block can be factored into one? Seems like a lot of code repetition.